beesdoo_website_eater/controllers/main.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2015-2016 Odoo S.A.
+# Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
```

Comment:
I think you're the only contributor of this code. so remove other people or entity same goes for the other files

Reply:
Done. :)